app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -59,25 +66,40 @@ def perform(claim_id)
       raise
     end
 
-    def self.send_failure_email(claim)
-      return unless Flipper.enabled?(:caregiver_use_va_notify_on_submission_failure)
-      return unless claim.parsed_form.dig('veteran', 'email')
-
-      parsed_form = claim.parsed_form
-      first_name = parsed_form.dig('veteran', 'fullName', 'first')
-      email = parsed_form.dig('veteran', 'email')
-      template_id = Settings.vanotify.services.health_apps_1010.template_id.form1010_cg_failure_email
-      api_key = Settings.vanotify.services.health_apps_1010.api_key
-      salutation = first_name ? "Dear #{first_name}," : ''
-
-      VANotify::EmailJob.perform_async(
-        email,
-        template_id,
-        { 'salutation' => salutation },
-        api_key
-      )
-
-      StatsD.increment("#{STATSD_KEY_PREFIX}submission_failure_email_sent")
+    class << self
+      def send_failure_email(claim)
+        unless can_send_failure_email?(claim)
+          StatsD.increment('silent_failure', tags: DD_ZSF_TAGS)
```

Comment:
Update this to increment the `silent_failure` since we may not necessarily have the email to send failure notification. We have existing monitors on the other metrics that are incremented previously.